src/interfaces/parity_coupon_message.ts

```diff
@@ -0,0 +1,10 @@
+import {Coupon} from 'types'
+
+export interface ParityCouponMessageProps {
```

Comment:
why not just add this to `types.ts` as a `type`?

we've long settled on being a `type` shop unless there's a compelling reason and I'm suspect of the need for a new directory that invites many interfaces to be created 🤔

Reply:
Fixed! https://github.com/eggheadio/egghead-next/pull/1018